messages/en.json

```diff
@@ -3,19 +3,19 @@
   "profile-form.include-business": "Include business fields",
   "profile-form.exclude-business": "Do not include business fields",
   "profile-form.save-changes": "Save changes",
-  "profile-form.field.firstName": "First name",
-  "profile-form.field.lastName": "Last name",
+  "profile-form.field.firstName": "First Name",
+  "profile-form.field.lastName": "Last Name",
   "profile-form.field.email": "E-mail",
   "profile-form.field.gender": "Gender",
-  "profile-form.field.custom-gender": "Custom gender",
+  "profile-form.field.custom-gender": "Custom Gender",
   "profile-form.field.document": "Document",
-  "profile-form.field.birthDate": "Birth date",
-  "profile-form.field.homePhone": "Home phone",
-  "profile-form.field.businessPhone": "Corporate phone",
-  "profile-form.field.corporateDocument": "Coporate document",
-  "profile-form.field.corporateName": "Corporate name",
-  "profile-form.field.stateRegistration": "State registration",
-  "profile-form.field.tradeName": "Trade name",
+  "profile-form.field.birthDate": "Birth Date",
+  "profile-form.field.homePhone": "Phone Number",
+  "profile-form.field.businessPhone": "Corporate Phone",
+  "profile-form.field.corporateDocument": "Corporate Document",
+  "profile-form.field.corporateName": "Corporate Name",
+  "profile-form.field.stateRegistration": "State Registration",
```

Comment:
Is there a reason for this change?

Reply:
In English we usually capitalize all the words in a title, and, in this case, these are the titles for the fields, so they should be capitalized. Also, some typos were corrected.

Reply:
Ok, but this should be done for the other branch aswell

Reply:
All right, thank you for the heads-up! I'll do that.

Reply:
Done, @salesfelipe ! The other PR is [here](https://github.com/vtex/profile-form/pull/129).